pkg/components/enterprise.go

```diff
@@ -57,12 +57,10 @@ var (
 
 	ComponentEckElasticsearch = component{
 		Version: "7.16.2",
-		Image:   "tigera/elasticsearch",
```

Comment:
Why is this removed?

Reply:
This was removed because this ComponentEckElasticsearch is only used to specify the version of Elasticsearch (in this case 7.16.2) so we can provide that to the ES operator. The ComponentElasticsearch has the image we use which can be found at line 73.